converter/src/test/java/gov/cms/qpp/conversion/model/validation/MeasureConfigsTest.java

```diff
@@ -1,57 +1,41 @@
 package gov.cms.qpp.conversion.model.validation;
 
-import com.fasterxml.jackson.databind.JsonMappingException;
-import org.junit.AfterClass;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.ExpectedException;
+import static com.google.common.truth.Truth.assertWithMessage;
 
-import java.io.IOException;
 import java.lang.reflect.Constructor;
 import java.util.List;
 
-import static com.google.common.truth.Truth.assertWithMessage;
-import static org.hamcrest.core.Is.isA;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
 
-public class MeasureConfigsTest {
-	@Rule
-	public ExpectedException thrown = ExpectedException.none();
+class MeasureConfigsTest {
 
-	@AfterClass
-	public static void resetMeasureConfiguration() {
+	@AfterAll
+	static void resetMeasureConfiguration() {
 		MeasureConfigs.setMeasureDataFile(MeasureConfigs.DEFAULT_MEASURE_DATA_FILE_NAME);
 	}
 
 	@Test
-	public void testGoodMeasureDataFile() {
+	void testGoodMeasureDataFile() {
 		MeasureConfigs.setMeasureDataFile("reduced-test-measures-data.json");
 		//no exception thrown
 	}
 
 	@Test
-	public void testNonExistingMeasureDataFile() {
-
-		//set-up
-		thrown.expect(IllegalArgumentException.class);
-		thrown.expectCause(isA(IOException.class));
-
-		//execute
-		MeasureConfigs.setMeasureDataFile("Bogus file name");
+	void testNonExistingMeasureDataFile() {
+		Assertions.assertThrows(IllegalArgumentException.class, () ->
```

Comment:
We are no longer asserting that `IOException` is the cause.

Reply:
Yeah, assertThrows returns a reference to the exception. You can use that to interrogate cause.

Reply:
@clydet sweet, didn't know that